### ISSUE TITLE
YouCompleteMe broke after switch from Vundle to vim-plug

### ISSUE THREAD

**Issue by DuncanBetts** [junegunn/vim-plug#549]
YouCompleteMe broke after switch, I was getting error:

> The ycmd server SHUT DOWN (restart with ':YcmRestartSer...re using it. Follow the instructions in the documentation.

I fixed this, by making this amendment to my vimrc:

> " Plug 'Valloric/YouCompleteMe', { 'do': './install.py --tern-completer' }

Then running :PlugClean, uncommenting the line, and then running :PlugInstall.

I thought this was worth reporting as I'm sure many vundle users also use YouCompleteMe and if switching to vim-plug causes this problem... also, if others are experiencing a similar issue they might find this post and save themselves a few minutes trying to figure out how to resolve it.

------------------------------

```
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Jun  2 2016 10:01:33)                                                                    
Included patches: 1-1868 
Full: http://pastebin.com/W2XiRQuv

```

- Type:
    - [x] Bug
    - [ ] Enhancement
    - [ ] Feature Request
    - [ ] Question
- OS:
    - [ ] All/Other
    - [x] Linux
    - [ ] OS X
    - [ ] Windows
- Vim:
    - [x] Terminal Vim
    - [ ] GVim
    - [ ] Neovim

**Comment by junegunn** [comment]
That's strange. Changing plugin manger should not affect the status of the plugin. How did you switch? Did you follow the instruction in https://github.com/junegunn/vim-plug/wiki/faq#migrating-from-other-plugin-managers?

/cc @vheon

**Comment by DuncanBetts** [comment]
I didn't find that page, but happened to follow similar steps.

Reading through that page (I hadn't seen it before now) I've spotted what caused my issue.
Everything was re-installed, because I didn't re-use the bundle directory, and when I attempted to re-compile the server, I'm pretty sure I was doing so in the bundle directory (I hadn't noticed the plugged folder was the new plugin directory, because bundle had identical content).
I'll go ahead and remove that directory now, thanks for your help :+1:
